examples/stm32f469_discovery/assert/main.cpp

```diff
@@ -51,10 +47,12 @@ int
 main()
 {
 	Board::initialize();
+	MODM_LOG_INFO << "\n=== RESTART ===\n" << modm::flush;
 
 	// trigger an IRQ with undefined handler
-	NVIC_EnableIRQ(RTC_Alarm_IRQn);
-	NVIC_SetPendingIRQ(RTC_Alarm_IRQn);
+	NVIC_EnableIRQ(OTG_FS_WKUP_IRQn);
+	NVIC_SetPendingIRQ(OTG_FS_WKUP_IRQn);
+
 
 	// trigger an out of memory
 	// we definitely don't have 32MB RAM on this board
```

Comment:
> 128-Mbit Quad-SPI NOR Flash

Only 16MB :wink:

Reply:
I know, hence the definitely. You could theoretically allocate 16MB, if you have an allocator that stores the information not inside the allocated (external=slow) memory, but in the internal `.fastdata` section.